docs/data-type-support.rst

```diff
@@ -0,0 +1,6 @@
+Data type support
+******************************************
+
+The hipRAND data type support is matching with the installed cuRAND data type support. The detailed description can be find at :doc:`rocRAND data type support page.<rocrand:data-type-support>`
```

Comment:
```suggestion
hipRAND's data type support is matching the installed cuRAND data type support. The detailed description can be found at :doc:`rocRAND's data type support page.<rocrand:data-type-support>`
```
Typo/wording

Also it feels weird to me when reading this, that it at first mentions, that it supports what cuRAND supports, but then links to rocRAND for more detailed information. Either link it to cuRAND's documentation, or at least mention that more details can be found in cuRAND's documentation, if we don't want to directly link there. Then we can still mention, that on AMD hardware it uses rocRAND as backend and link to rocRAND's data type support page.

Reply:
@MKKnorr please take over the PR. It was your task originally :)

Reply:
Well, the rocRAND one, but yeah, sure :)